usermanager/src/test/java/org/pesmypetcare/usermanager/clients/pet/PetCollectionsManagerClientTest.java

```diff
@@ -0,0 +1,644 @@
+    private PetCollectionsManagerClient client = new PetCollectionsManagerClient();
+
+    @Rule
+    public MockitoRule mockitoRule = MockitoJUnit.rule();
+
+    @Before
+    public void setUp() {
+        // Corregir exercise per ficar coordinates
+        exerciseCollectionJson = new StringBuilder("[\n"
+                + "  {\n"
+                + "    \"body\":{\n"
+                + "      \"name\":\"Planking\",\n"
+                + "      \"coordinates\":[\n"
+                + "        {\n"
+                + "          \"latitude\":84,\n"
+                + "          \"longitude\":58\n"
+                + "        },{\n"
+                + "          \"latitude\":34,\n"
+                + "          \"longitude\":58\n"
+                + "        },{\n"
+                + "          \"latitude\":21,\n"
+                + "          \"longitude\":58\n"
+                + "        }\n"
+                + "      ],\n"
+                + "      \"description\":\"Staying horizontal over a plain surface\",\n"
+                + "      \"endDateTime\":\"2000-01-08T15:20:30\"\n"
+                + "    },\n"
+                + "    \"key\":\"1990-01-08T15:20:30\"\n"
+                + "  },{\"body\":{\n"
+                + "      \"name\":\"Planking\",\n"
+                + "      \"coordinates\":[\n"
+                + "        {\n"
+                + "          \"latitude\":84,\n"
+                + "          \"longitude\":58\n"
+                + "        },{\n"
+                + "          \"latitude\":34,\n"
+                + "          \"longitude\":58\n"
+                + "        },{\n"
+                + "          \"latitude\":21,\n"
+                + "          \"longitude\":58\n"
+                + "        }\n"
+                + "      ],\n"
+                + "      \"description\":\"Staying horizontal over a plain surface\",\n"
+                + "      \"endDateTime\":\"2000-01-08T15:20:30\"\n"
+                + "    },\n"
+                + "    \"key\":\"1995-01-08T15:20:30\"\n"
+                + "  },{\"body\":{\n"
+                + "      \"name\":\"Planking\",\n"
+                + "      \"coordinates\":[\n"
+                + "        {\n"
+                + "          \"latitude\":84,\n"
+                + "          \"longitude\":58\n"
+                + "        },{\n"
+                + "          \"latitude\":34,\n"
+                + "          \"longitude\":58\n"
+                + "        },{\n"
+                + "          \"latitude\":21,\n"
+                + "          \"longitude\":58\n"
+                + "        }\n"
+                + "      ],\n"
+                + "      \"description\":\"Staying horizontal over a plain surface\",\n"
+                + "      \"endDateTime\":\"2000-01-08T15:20:30\"\n"
+                + "    },\n"
+                + "    \"key\":\"1998-01-08T15:20:30\"\n"
+                + "  }\n"
+                + "]");
+        exerciseDataJson = new StringBuilder("{\n"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String "{\n" appears 8 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Libraries/pullRequest?prid=5549434)